src/components/Type/Type.module.scss

```diff
@@ -107,10 +107,9 @@
 }
 
 .Body {
-  $font-size: 19;
-  $line-height: 24;
-  font-size: $font-size * 1px;
-  line-height: $line-height / $font-size;
+  --base-line-height: 24;
+  font-size: calc(var(--base-font-size) * 1px);
+  line-height: calc(var(--base-line-height) / var(--base-font-size));
```

Comment:
i believe i had issues mixing some combination of calc, scss variables, and css variables in some eds-> monorepo or cms deployment process at some point. i would recommend being careful about this, and possibly trying to jump through all the hoops to pull this into cms and make sure `yarn build` runs, as well as pulling this into monorepo and deploying it to a sandbox or staging server to avoid the crazy 'scss comments' blocker that we had about a month ago. 

alternatively you could just do `font-size: 19px; line-height: 1.2632;` but where's the fun in that 😛

Reply:
Those are defined in 

`src/components/Type/Type.scss` and the index: `src/components/index.scss` in turn imports that into the global css that results in design-system.css so I don't see this being an issue without all of our globals being an issue.